analyzers/tests/SonarAnalyzer.Test/TestCases/SymbolicExecution/Roslyn/HashesShouldHaveUnpredictableSalt.CSharp10.cs

```diff
@@ -4,7 +4,8 @@ public class Sample
 {
     public void Examples(byte[] passwordBytes)
     {
-        (var shortSalt, int a) = (new byte[15], 42);
+        int a;
+        (var shortSalt, a) = (new byte[15], 42);
```

Comment:
Why?

Reply:
To make it actually require C# 10 (mixing assignment and declaration in deconstruction). However, since I removed the comment in the test file, this change does not need to be in this PR. I'll revert it.